release-team/role-handbooks/enhancements/README.md

```diff
@@ -131,60 +131,58 @@ It is important that this process be followed and documentation remain up-to-dat
+    - Update `Enhancement Status` field for this Enhancement in the Enhancement Tracking Board (`Tracked` or `At Risk`)
+- Reach out to each [SIG on Slack](https://github.com/kubernetes/community/blob/master/sig-list.md) tagging one or more chairs or technical leads to communicate the Enhancements statuses. Example [communication templates](#sig-outreach-templates) can be found below.
+- Start syncing with Communications Team on giving an induction what's coming up for the release.
 - Send an email to [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) that Enhancement freeze is coming and share current Enhancements status. Examples [1](https://groups.google.com/g/kubernetes-dev/c/-nTNtBBHL2Y/m/WfNzb_E1EAAJ).
+- Provide updates during release team meetings
+  - Use the `KEPs by Stage` insight from the Enhancement Tracking Board ([example](https://github.com/orgs/kubernetes/projects/98/insights/3))
 
-#### Enhancements Freeze (Week 3)
+#### Enhancements Freeze (Week 5)
 
 - On Freeze day, send an email to [Kubernetes-Dev](https://groups.google.com/a/kubernetes.io/g/dev) that freeze has happened and upcoming key dates. Examples [1](https://groups.google.com/g/kubernetes-dev/c/JDM7bNKvhqQ/m/8S7BXtXPBQAJ).
-- Remove [SIG Leads Google Group][sig-leads-group]'s access to the `KEP Collection` tab on the enhancement collection sheet.
-- Remove any enhancements that failed to meet the criteria by the Enhancement freeze deadline.
-  - Set their status in the sheet to `Removed from Milestone` and use the `Enhancements` -> `Remove Enhancements from Milestone` menu option to move them over to the `Removed from milestone` tab.
-  - Remove the milestone and change `tracked/yes` label to `tracked/no` on the enhancement issue.
+- Remove any Enhancements that failed to meet the criteria by the Enhancement freeze deadline.
+  - Set their **Enhancement Status** in the board to `Removed from Milestone`.
+  - Remove the milestone and change `tracked/yes` label to `tracked/no` on the Enhancement Issue.
 - Clean up Enhancements issues by removing milestone from the enhancements that have not opted-in and make sure that number of in-tree open issues with current milestone matches number of opted-in enhancements.
 - Any enhancements removed from the milestone will now require an exception. As exception requests come in, discuss each with the Release Lead (and Shadows) to arrive at an approve/reject decision.
   - Create an exception file in the Release for exceptions Example [1](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/exception.yaml).
-  - If a previously removed Enhancement has had their exception Approved, set their status to `Tracked` and use the `Enhancements` -> `Track Removed Enhancements` menu option to move it back to the `Enhancements` and `Docs` tabs.
+  - If a previously removed Enhancement has had their exception Approved, set their **Enhancement Status** to `Tracked` in the board.
```

Comment:
```suggestion
  - If a previously removed Enhancement has had their exception Approved, set their **Enhancement Status** to `Tracked` in the enhancements board.
```

Reply:
maybe this extra explicitly is not needed (feel free to drop this comment)

Reply:
updated